README.md

```diff
@@ -1,5 +1,22 @@
-# Introduction to Helm 
+# Introduction to Helm v2
```

Comment:
nit and you can decide but I think no need to add v3 in the title here...the name of repo shows that it's about v2. In the introduction there is nothing specific about v2 so I would leave the title of this Intro section as is.

Reply:
Agree, removed